corehq/apps/export/models/new.py

```diff
@@ -721,6 +753,37 @@ def merge(cls, one, two):
         item.options = options
         return item
 
+    def split_header(self, header, ignore_unspecified_options):
+        header_template = header if '{option}' in header else u"{name} | {option}"
+        headers = []
+        for option in self.options:
+            headers.append(
```

Comment:
It's possible to render template like strings using Django template engine like below, the logic is somewhat easier to grasp that way.
```python
In [29]: from django.template import Template, Context

In [30]: header = "{% for x in options %}{{ x }}{% endfor %}"

In [31]: Template(header).render(Context({"options":[1,3]}))
Out[31]: u'13'
```
Though, I am not sure if it's favorable